sdk/attestation/azure-security-attestation/README.md

```diff
@@ -306,7 +291,7 @@ Most Attestation service operations will raise exceptions defined in [Azure Core
 
 ```python
     try:
-    response = attest_client.attest_sgx_enclave(
+    response, _ = attest_client.attest_sgx_enclave(
             quote,
             runtime_data=AttestationData(runtime_data, is_json=False))
     except HttpResponseError as ex:
```

Comment:
These lines (and the rest of the snippet, which I apparently can't include in a review comment 🤔) can have their indentation removed